python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -601,6 +608,7 @@ def run(self, exposure=None, selectSources=None, templateExposure=None, template
 
         if self.config.doSubtract:
             if self.config.doScaleTemplateVariance:
+                self.log.info("Rescaling template variance")
                 templateVarFactor = self.scaleVariance.run(
                     templateExposure.getMaskedImage())
                 self.metadata.add("scaleTemplateVarianceFactor", templateVarFactor)
```

Comment:
Please add a log message stating the template variance scaling that is being used.